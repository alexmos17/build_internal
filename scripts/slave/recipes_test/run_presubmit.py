# Copyright (c) 2013 The Chromium Authors. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.

"""Test inputs for recipes/run_presubmit.py"""

def blink_bare_test(api):
  return {
    'factory_properties': {'repo_name': 'blink_bare'},
    'build_properties': api.tryserver_build_properties(
      root='src/third_party/WebKit'
    ),
  }

def blink_test(api):
  return {
    'factory_properties': {'repo_name': 'blink'},
    'build_properties': api.tryserver_build_properties(
      root='src/third_party/WebKit'
    ),
  }

def tools_build_test(api):
  return {
    'factory_properties': {'repo_name': 'tools_build'},
    'build_properties': api.tryserver_build_properties(
      root='src'
    ),
  }

def chromium_test(api):
  return {
    'factory_properties': {'repo_name': 'chromium'},
    'build_properties': api.tryserver_build_properties(
      root='src'
    ),
  }