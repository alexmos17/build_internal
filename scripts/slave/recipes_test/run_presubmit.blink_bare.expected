[
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/gclient", "config", "--spec", "solutions = [{'url': 'svn://svn-mirror.golo.chromium.org/blink/trunk', 'safesync_url': '', 'managed': True, 'name': 'blink', 'deps_file': ''}]\n"],
   "name": "gclient setup",
   "seed_steps": ["gclient setup", "gclient clean", "gclient sync"]
  },
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/gclient", "revert", "--nohooks"],
   "name": "gclient clean",
   "seed_steps": []
  },
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/gclient", "sync", "--nohooks"],
   "name": "gclient sync",
   "seed_steps": []
  },
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/apply_issue", "-r", "[SLAVE_BUILD_ROOT]/blink/", "-i", 12853011, "-p", 1, "-s", "https://chromiumcodereview.appspot.com", "-e", "commit-bot@chromium.org"],
   "name": "apply_issue"
  },
  {
   "cmd": ["[DEPOT_TOOLS_ROOT]/presubmit_support.py", "--root", "[SLAVE_BUILD_ROOT]/blink/", "--commit", "--verbose", "--verbose", "--issue", 12853011, "--patchset", 1, "--skip_canned", "CheckRietveldTryJobExecution", "--skip_canned", "CheckTreeIsOpen", "--skip_canned", "CheckBuildbotPendingBuilds", "--rietveld_url", "https://chromiumcodereview.appspot.com", "--rietveld_email", "", "--rietveld_fetch"],
   "name": "presubmit"
  }
]